### ISSUE TITLE
Need to stop behavior ; Extra default style added to each tag , and commentize none HTML tags like </#list>

### ISSUE THREAD

**Issue by shareefhiasat** [GrapesJS/preset-newsletter#42]
Hi

**Explanation**
Am using news-letter-preset its cooool

**Problem**
I have un desired behavior

**More Info About The Problem**
Check below two links and try do export can you tell the difference here is a screen shot :)

[with gjs-preset-newsletter plugin https://jsfiddle.net/shareefhiasat/Lup78L17/5/ ](https://jsfiddle.net/shareefhiasat/Lup78L17/5/ )

[with gjs-blocks-basic plugin https://jsfiddle.net/shareefhiasat/4q3kxthj/1/ ](https://jsfiddle.net/shareefhiasat/4q3kxthj/1/ )

**So What ?**
Its the same fiddle ; the only difference is the plugin

**What is Expected**
I expect no extra attributes like style to be added to the tag in my content 

![image](https://user-images.githubusercontent.com/1712274/40918645-90a7d9aa-680f-11e8-897f-089bd6a237ad.png)

**UPDATE**

I could stop the style from appearing but there is extra double quotations now check this
( How like this by commenting css , ( i have no idea what is the impact of this so please advise )  )
![image](https://user-images.githubusercontent.com/1712274/40963553-95886168-68b1-11e8-867b-fe5f5268f07e.png)

**Now Another Problem**

![image](https://user-images.githubusercontent.com/1712274/40934048-33535662-683c-11e8-8e00-045ef569c844.png)

These double quotations is wrong in free marker syntax render in back end .

@artf can you advise thanks

In Short 
Original code that must stay as is
`<#assign currentDate=. now></#assign>`

Becomes 

`<#assign currentDate="." now="">`

And this is wrong , throws exception , not correct syntax !

**UPDATE**

After alot of more tries, i decided to try adding it as component now its ok 
after i commented this line
[https://github.com/artf/grapesjs-preset-newsletter/blob/601ad471b29c2ffb08bc07ad9c8ecc0b6064dd21/src/openExportCommand.js#L44](https://github.com/artf/grapesjs-preset-newsletter/blob/601ad471b29c2ffb08bc07ad9c8ecc0b6064dd21/src/openExportCommand.js#L44)

And replace with

`codeViewer.setContent(tmpl);`

But still there is a problem which is `</#list>` dose not appear on screen or source of HTML why.

https://jsfiddle.net/3xhqkc96/12/

@rgcarrasqueira  @santiph  @artf

**Comment by shareefhiasat** [comment]
The parser 
here
[https://github.com/artf/grapesjs/blob/f74791da68b25d1037502a2ad06093da29e25830/src/parser/index.js#L49](https://github.com/artf/grapesjs/blob/f74791da68b25d1037502a2ad06093da29e25830/src/parser/index.js#L49)

Discardes my end tags for the free marker like `</#list> </#assign>`

**Comment by shareefhiasat** [comment]
Hi all ; @rgcarrasqueira @santiph @artf  any chance to advise here thanks
I ended up now creating component for each part and resave  the template that way it is ok but lot of work, i believe there is easier way please help.

Thanks
